packages/opentelemetry-sdk-trace-base/src/SpanProcessor.ts

```diff
@@ -35,6 +35,15 @@ export interface SpanProcessor {
    */
   onStart(span: Span, parentContext: Context): void;
 
+  /**
+   * Called when a {@link Span} is ending, if the `span.isRecording()`
+   * returns true.
+   * @param span the Span that is ending.
+   *
+   * @experimental This method is experimental and subject to change
```

Comment:
we could be a bit more explicit:
```suggestion
   * @experimental This method is experimental and may break in minor versions of this package
```